### PR TITLE
Add script covering case when TTS.Speak gets WARNINGS and UI.PerformA…

### PR DESCRIPTION
**Pull request by vvvakulenko**
…udioPassThru gets UNSUPPORTED_RESOURCE from HMI
Script is a part of TS #666

### CODE REVIEW

**Comment by vvvakulenko** [comment]
@istoimenova Please review

File: test_scripts/API/PerformAudioPassThru/027_ATF_UI_IsReady_no_resp_PerfAudioPassThru_UI_PerfAudioPassThru_UNSUP_RES.lua [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_initHMI')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+function Test:Precondition_Check_audioPassThruIcon_Existence()
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/027_ATF_UI_IsReady_no_resp_PerfAudioPassThru_UI_PerfAudioPassThru_UNSUP_RES.lua]: @vvvakulenko, this function is not needed in scope of the requirement. If you leave it, move it together with PutFile(see comment above) after application is registered.
If you remove it, don't forget to remove reference of testCasesForPerformAudioPassThru

File: test_scripts/API/PerformAudioPassThru/027_ATF_UI_IsReady_no_resp_PerfAudioPassThru_UI_PerfAudioPassThru_UNSUP_RES.lua [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"},"PerformAudioPassThru")
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/connecttest_initHMI')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/027_ATF_UI_IsReady_no_resp_PerfAudioPassThru_UI_PerfAudioPassThru_UNSUP_RES.lua]: @vvvakulenko, please move the function after application is registered.

Comment (vvvakulenko) [inline test_scripts/API/PerformAudioPassThru/027_ATF_UI_IsReady_no_resp_PerfAudioPassThru_UI_PerfAudioPassThru_UNSUP_RES.lua]: @istoimenova , updated in commit ac2a9a3952995ecaa7ef888132df6cc28982465d

File: test_scripts/API/PerformAudioPassThru/027_ATF_UI_IsReady_no_resp_PerfAudioPassThru_UI_PerfAudioPassThru_UNSUP_RES.lua (hunk 1 above)
Comment (vvvakulenko) [inline test_scripts/API/PerformAudioPassThru/027_ATF_UI_IsReady_no_resp_PerfAudioPassThru_UI_PerfAudioPassThru_UNSUP_RES.lua]: @istoimenova , updated in commit ac2a9a3952995ecaa7ef888132df6cc28982465d

**Comment by vvvakulenko** [comment]
@istoimenova , Please see updates in commit ac2a9a3952995ecaa7ef888132df6cc28982465d

File: test_scripts/API/PerformAudioPassThru/027_ATF_UI_IsReady_no_resp_PerfAudioPassThru_UI_PerfAudioPassThru_UNSUP_RES.lua [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+-- ToDo (vvvakulenko): remove after issue "ATF does not stop HB timers by closing session and connection" is resolved
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local testCasesForUI_IsReady = require('user_modules/IsReady_Template/testCasesForUI_IsReady')
+local testCasesForPerformAudioPassThru = require('user_modules/shared_testcases/testCasesForPerformAudioPassThru')
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/027_ATF_UI_IsReady_no_resp_PerfAudioPassThru_UI_PerfAudioPassThru_UNSUP_RES.lua]: @vvvakulenko, testCasesForPerformAudioPassThru is not use, please remove the row.

**Pull request merged**